task/09-functions-n-closures-tasks.js

```diff
@@ -47,7 +55,11 @@ function getComposition(f,g) {
  *
  */
 function getPowerFunction(exponent) {
-    throw new Error('Not implemented');
+    let func = function(x){
+        return Math.pow(x,exponent);
+    }
+    return func
```

Comment:
You can write it like:
return x => Math,pow(x, exponent)
Same, but much shorter :)